sdk/src/worker.ts

```diff
@@ -0,0 +1,145 @@
+import { initializeWasm, initThreadPool, ProgramManager, PrivateKey, verifyFunctionExecution } from "./browser";
+import { AleoKeyProvider, AleoKeyProviderParams} from "./function-key-provider";
+import { expose } from "comlink";
+
+await initializeWasm();
+await initThreadPool(10);
+
+const defaultHost = "https://vm.aleo.org/api";
+const keyProvider = new AleoKeyProvider();
+const programManager = new ProgramManager(
+    defaultHost,
+    keyProvider,
+    undefined
+);
+
+keyProvider.useCache(true);
+
+// self.postMessage({
+//     type: "ALEO_WORKER_READY",
+// });
+
+export interface WorkerAPI {
+    additionalFunction1: () => Promise<string>;
+    additionalFunction2: (data: any) => Promise<any>;
+}
```

Comment:
Does this conflict with the definition immediately following/do we need this one?

Reply:
Oops sorry left this by accident too, removed.